docs/tools/vega-wallet/cli-wallet/latest/guides/isolate-keys.md

```diff
@@ -0,0 +1,250 @@
+---
+sidebar_position: 4
+title: Isolate keys
+hide_title: false
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+If you want to move a key pair from a standard wallet into a wallet that can only sign and verify transactions, use the `key isolate` command. Isolating a key pair will extract a chosen key pair from one wallet and create an isolated wallet with only that key pair.
+
+This optional step creates an extra layer of security, particularly for validators and others who are concerned about their wallets being compromised. 
+
+An isolated wallet can only contain a single key pair that has been stripped from its cryptographic node. A wallet without a cryptographic node, which is used to create keys, minimises the impact of that wallet being compromised, as it makes it impossible to retrieve or generate keys. 
```

Comment:
```suggestion
An isolated wallet can only contain a single key pair that has been stripped from its cryptographic node. A wallet without a cryptographic node minimises the damages if the wallet has been compromised, as it makes it impossible to generate keys or retrieve any other keys than isolated one. 
```

It's very badly worded, but I think sentence is too heavy and convoluted. We should find a better way to explain why not having this node is beneficial to security.

Reply:
An isolated wallet can only contain a single key pair that has been stripped from its cryptographic node. If a wallet without a cryptographic node is compromised, the damage is minimised as it is impossible to generate keys or retrieve any keys other than the isolated one. 


How about that ^ @ValentinTrinque?

Reply:
It's honey to my ears.